### ISSUE TITLE
d3.sliderRight() not display correctly

### ISSUE THREAD

**Issue by iosonosempreio** [johnwalley/d3-simple-slider#110]
Track line not covering all the scale when slider created with `d3.sliderRight()`.
Problem solves if created with `d3.sliderLeft()` as in the provided example.

```
import * as d3 from 'd3';
import * as d3sl from 'd3-simple-slider';

var sliderVertical = d3sl
      .sliderRight()
      .min(Utilities.zoomOptions.minScale)
      .max(Utilities.zoomOptions.maxScale)
      .default(1)
      .height(slider_height-30)
      .fill('#2E4F4F')
      .tickValues([0.2, 1, 2, 3])

    var gVertical = d3
      .select('#zoom-slider')
      .append('svg')
      .attr('width', slider_width)
      .attr('height', slider_height)
      .append('g')
      .attr('transform', 'translate('+(slider_width/2)+',15)');
  
    gVertical.call(sliderVertical);
```

The above code generates this, **which is not correct**:
<img width="83" alt="Screen Shot 2020-03-17 at 21 39 33" src="https://user-images.githubusercontent.com/7115395/76916756-d28c9380-6897-11ea-8b59-2e86a0c8c6ee.png">

```
import * as d3 from 'd3';
import * as d3sl from 'd3-simple-slider';

var sliderVertical = d3sl
      .sliderLeft()
      .min(Utilities.zoomOptions.minScale)
      .max(Utilities.zoomOptions.maxScale)
      .default(1)
      .height(slider_height-30)
      .tickValues([0.2, 1, 2, 3])

    var gVertical = d3
      .select('#zoom-slider')
      .append('svg')
      .attr('width', slider_width)
      .attr('height', slider_height)
      .append('g')
      .attr('transform', 'translate('+(slider_width/2)+',15)');
  
    gVertical.call(sliderVertical);
```

The second block, generates the following, which is correct, but I would love to have numbers on the opposite side:
<img width="90" alt="Screen Shot 2020-03-17 at 21 41 16" src="https://user-images.githubusercontent.com/7115395/76916828-18495c00-6898-11ea-9394-498ab9acc8cf.png">

Could it be a problem related to how I import the library?

**Comment by johnwalley** [comment]
Thanks for reporting this. It also affects `sliderTop`. I have published version `1.5.5` which should fix the issue but do let me know if it does not!

**Comment by johnwalley** [comment]
Please re-open if this is still an issue. Closing on the assumption it is now fixed.

**Comment by aidan-epperly** [comment]
Not sure if this is the same issue or not, but I am having a similar problem. I am trying to create and then hide a slider and then bring it back again. I do this by first creating a slider
``` js
const slider = d3
    .sliderRight()
    .domain([0,3])
    .step(1)
    .tickFormat(d => {
        return options[Math.round(d)];
    })
    .ticks(3)
    .value(sliderValue)
    .height(100)
    .on('onchange', val => {
       sliderValue = val;
       val = options[Math.round(val)];
       data = reformatData(obj, val);
       root = partition(data);
       chart.select('#licenseCenter').remove();
       update();
    });
```
Which yields:
![image](https://user-images.githubusercontent.com/29114346/86843564-95712a80-c05b-11ea-99f0-c5495a1a3985.png)

Then, later I use this if/else to create/delete the slider
``` js
if (o.depth == 0) {
    console.debug('beep');
    sliderBox.selectAll('g').remove();
    sliderBox.call(slider);
} else {
    console.debug('boop');
    sliderBox.selectAll('g').remove();
}
```
Which yields:
![image](https://user-images.githubusercontent.com/29114346/86843848-03b5ed00-c05c-11ea-8279-595579c0ec3e.png)

Please let me know if you have any ideas on fixing it. Code for the project lives [here](https://github.com/aidan-epperly/llnl.github.io/blob/slider-adjustments/js/explore/sunburst_licenses.js).

**Comment by johnwalley** [comment]
Thanks for taking the time to report this @aidan-epperly. I'll take a look and get back to you!

**Comment by johnwalley** [comment]
@aidan-epperly I've published v1.7.2 which I should address your issue. If you could test it out and let me know I'd be grateful. 👍

**Comment by aidan-epperly** [comment]
@johnwalley Yes, that fixes it! Thank you for the help.
